OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -21,10 +21,9 @@
 
 namespace OpenRA.Mods.Cnc.Traits
 {
-	public class MinelayerInfo : ITraitInfo
+	public class MinelayerInfo : ITraitInfo, Requires<RearmableInfo>
```

Comment:
Do we want to force that? Vultures and Spider Mines from Star Craft comes to my mind, which every vulture has 3 mines that cannot be rearmed.

Reply:
This trait isn't really appropriate for that.  All the logic in this trait relates to setting the large deployment area and automatic reloading.